src/example-button.module/fields.json

```diff
@@ -0,0 +1,230 @@
+[
+  {
+    "label": "Button text",
+    "name": "button_text",
+    "type": "text",
+    "required": true,
+    "default": "Click here"
+  },
+  {
+    "label": "Button link",
+    "name": "button_link",
+    "type": "link",
+    "supported_types": ["EXTERNAL", "CONTENT", "FILE", "EMAIL_ADDRESS", "BLOG"],
+    "default": {
+      "url": {
+        "href": "",
+        "type": "EXTERNAL"
+      }
+    }
+  },
+  {
+    "label": "Styles",
+    "name": "styles",
+    "type": "group",
+    "tab": "STYLE",
+    "children": [
+      {
+        "label": "Text",
+        "name": "text",
```

Comment:
nit/question: since this is an example, would it make sense to model "group_" naming to encourage better naming conventions?

Reply:
Yup! Good call - just added that in there now